www/js/survey/multilabel/ModeSelectMenu.jsx

```diff
@@ -0,0 +1,35 @@
+/* A scrollable popover menu for choosing the mode when labeling trips */
+
+import React, { } from "react";
+import { angularize } from "../../angular-react-helper";
+import { func } from "prop-types";
+import { ScrollView } from "react-native";
+import { Menu } from "react-native-paper";
+import { useTranslation } from "react-i18next";
+
+const ModeSelectMenu = ({ anchor, visible, inputParams, chooseFn }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Menu visible={visible} anchor={anchor}>
+      <ScrollView>
+        {inputParams?.MODE?.options.map((mode) => {
+          return (
+            <Menu.Item
+              key={mode}
+              onPress={() => chooseFn('MODE', mode)}
+              title={t(mode.text)}
```

Comment:
@Abby-Wheelis Then, wherever a translation needs to happen, we'll just pass the translation ID to `t()`

Another example: `t('general-settings.are-you-sure')` will return `"Are you sure?"` if the locale is English, or `"¿Estás seguro?"` if the locale is Spanish.